### ISSUE TITLE
 answers for exercise 2  are weird..

### ISSUE THREAD

**Issue by ghicheon** [qiskit-community/may4_challenge_exercises#8]
I'm sure my answer is correct. 
And it's not a difficult problem. it's just about selecting the similar graph, isn't it?
Is there anyone who is suffering from this one?
Am I missing something important?

**Comment by Anand12568** [comment]
yes same iam experiencing

**Comment by abhik-99** [comment]
https://github.com/qiskit-community/may4_challenge_exercises/issues/9#issuecomment-625495667

- Hope this helps. Basically, after getting the mitigated results, you can have a better understanding of how the histogram plot would look for a perfect quantum computer.

**Comment by ghicheon** [comment]
thanks for your comment!
I think I understood the concept of the mitigation.
I understand the video https://www.youtube.com/watch?v=yuDxHJOKsVA 
But the grader is still saying the answer is wrong.
It's one of the following probabilities.
---------------------------------------------------------------------
1. I didn't understand actually and couldn't select the currect answer.
**2. the grader is not working.**
----------------------------------------------------------------------

**Comment by ghicheon** [comment]
helpful information was written by ethiopianqubit  on youtube.

-------------------------------------------------------------------------------------------------
https://www.youtube.com/watch?v=3oNXkCZ9odQ&t=1s
Hi! It's quite normal to see some small counts after measurement error mitigation. This is why we designed for exercise 2 to ask the error-free counts.
-------------------------------------------------------------------------------------------------
Wow...They're asking the "error-free counts".
I didn't notice it. it's confusing..
the bottom line is that I didn't understand the question :D   hahaha

I just leave this comments for someone who is not having a good time with exercise 2.

Thanks all !!
I close this issue.
